governance/interface.go

```diff
@@ -29,6 +29,8 @@ import (
 type Engine interface {
 	HeaderEngine
 	ReaderEngine
+	HeaderGov() HeaderEngine
+	ContractGov() ReaderEngine
```

Comment:
When are these used?

Reply:
@blukat29 ContractGov() is for test (link below), HeaderGov() does not have any use.
https://github.com/ian0371/klaytn/blob/4a28f7e4a45977f8a9ee82435967e94de3b981cd/tests/gov_contract_test.go#L95